Cos3rd.java

```diff
@@ -1,13 +1,29 @@
 class Cos3rd extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
```

Comment:
Overall code looks good, there are just minor differences in the comments and in the calculation part but it still computes the same answer.